Modules/LearningSequence/classes/class.ilObjLearningSequenceGUI.php

```diff
@@ -156,6 +156,19 @@ public function executeCommand()
 
         //exit real early for LP-checking.
         if ($cmd === LSControlBuilder::CMD_CHECK_CURRENT_ITEM_LP) {
+            if (isset($_GET[LSControlBuilder::PARAM_LP_CURRENT_ITEM_OBJID])) {
+                $current_obj_obj_id = (int) $_GET[LSControlBuilder::PARAM_LP_CURRENT_ITEM_OBJID];
+
+                print (int) ilLPStatus::_lookupStatus(
+                    $current_obj_obj_id,
+                    $this->user->getId(),
+                    true
+                );
+                exit;
```

Comment:
This introduces a new dicto violation.

For ILIAS >= 6 I suggest to use `\ILIAS\HTTP\GlobalHttpState::saveResponse` and `\ILIAS\HTTP\GlobalHttpState::close`

Reply:
Good catch @mjansenDatabay, thx a lot!!

Reply:
Good spotting; it's not that new, though, it rather perpetuates the old one ;)
I'll have a go a \ILIAS\HTTP\GlobalHttpState with trunk/6, thank you for pointing that out.